js/backup.js

```diff
@@ -41,24 +43,36 @@
     return object;
   }
 
-  function createOutputStream(fileWriter) {
+  function createOutputStream(writer) {
     var wait = Promise.resolve();
-    var count = 0;
     return {
       write: function(string) {
-        var i = count++;
         wait = wait.then(function() {
           return new Promise(function(resolve, reject) {
-            fileWriter.onwriteend = resolve;
-            fileWriter.onerror = reject;
-            fileWriter.onabort = reject;
-            fileWriter.write(stringToBlob(string));
+            writer.once('error', reject);
```

Comment:
It's not at all compatible. The branches are absolutely divergent at this point. I've even removed all migration-related stuff from this branch, since Electron has no need for that experience.